yarn-project/pxe/src/synchronizer/synchronizer.test.ts

```diff
@@ -16,67 +17,59 @@ describe('Synchronizer', () => {
   let aztecNode: MockProxy<AztecNode>;
   let database: PxeDatabase;
   let synchronizer: TestSynchronizer;
-  let roots: Record<MerkleTreeId, Fr>;
-  let blockHeader: BlockHeader;
   let jobQueue: SerialQueue;
+  const initialSyncBlockNumber = 3;
+  let headerBlock3: Header;
 
   beforeEach(async () => {
-    blockHeader = BlockHeader.random();
-    roots = {
-      [MerkleTreeId.CONTRACT_TREE]: blockHeader.contractTreeRoot,
-      [MerkleTreeId.NOTE_HASH_TREE]: blockHeader.noteHashTreeRoot,
-      [MerkleTreeId.NULLIFIER_TREE]: blockHeader.nullifierTreeRoot,
-      [MerkleTreeId.PUBLIC_DATA_TREE]: blockHeader.publicDataTreeRoot,
-      [MerkleTreeId.L1_TO_L2_MESSAGE_TREE]: blockHeader.l1ToL2MessageTreeRoot,
-      [MerkleTreeId.ARCHIVE]: blockHeader.archiveRoot,
-    };
+    headerBlock3 = makeHeader(Math.floor(Math.random() * 1000), initialSyncBlockNumber);
 
     aztecNode = mock<AztecNode>();
     database = new KVPxeDatabase(await AztecLmdbStore.openTmp());
     jobQueue = new SerialQueue();
     synchronizer = new TestSynchronizer(aztecNode, database, jobQueue);
   });
 
-  it('sets tree roots from aztec node on initial sync', async () => {
+  it('sets header from aztec node on initial sync', async () => {
     aztecNode.getBlockNumber.mockResolvedValue(3);
```

Comment:
Should this value also be mocked with the `initialSyncBlockNumber`?

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/4154/commits/4d31d69113684d7df3db4eaa73bd00e90c32cd95